tests/fn_boundary.json

```diff
@@ -388,6 +454,30 @@
         {
           "id": "o25",
           "decimal": null
+        },
+        {
+          "id": "o26",
+          "decimal": 1.58700000
```

Comment:
JS removes all zeros in the end, so this value looks like 1.587, and lowBoundary will return 1.587, although they are both should be 1.58700000.